lib/react_on_rails/utils.rb

```diff
@@ -25,7 +25,14 @@ def self.running_on_windows?
     end
 
     def self.rails_version_less_than(version)
-      Gem::Version.new(Rails.version) < Gem::Version.new(version)
+      @rails_version_less_than ||= {}
+      @rails_version_less_than[version] = begin
+        Gem::Version.new(Rails.version) < Gem::Version.new(version)
+      end
```

Comment:
True. I'm gonna fix it, thanks for the review. Can you tell me why my test passes and so, is not correct? https://github.com/shakacode/react_on_rails/pull/812/files#diff-ad2d9ab6a9a22bf3f00405aba4094cbaR38